### DISCUSSION TITLE
Webinstaller issue

### DISCUSSION THREAD

**Discussion by moatazali** [pear-devs/pear-desktop#1814] [Q&A] [answered]
Web Installer issue with windows policy, can we get to download the full install file ?

**Comment by JellyBrick** [comment, marked as answer]
Please download the latest version.
and, if you want to install the YTMD without a network connection, read this manual. 
https://github.com/th-ch/youtube-music#how-to-install-without-a-network-connection-in-windows

**Reply by moatazali** [reply]
Perfect thank you
